src/views/studio/EditVideoSheet/EditVideoForm/EditVideoForm.tsx

```diff
@@ -149,7 +154,28 @@ export const EditVideoForm: React.FC<EditVideoFormProps> = ({
 
     setFileSelectError(null)
     reset(tabData)
-  }, [selectedVideoTab, cachedSelectedVideoTabId, forceReset, reset, tabDataLoading, tabData, updateSelectedVideoTab])
+
+    if (selectedVideoTabCachedDirtyFormData) {
+      // allow a render for the form to reset first and then set fields dirty
+      setTimeout(() => {
```

Comment:
It'd be best if we can avoid the timeout to not have the action bar sliding up and down for one render. Let's continue discussion on rocket chat

Reply:
Let's leave it for now, but let's create an issue to capture this behaviour